test/v3/services/SubmitFinalDeclarationServiceSpec.scala

```diff
@@ -43,7 +43,7 @@ class SubmitFinalDeclarationServiceSpec extends ServiceSpec {
     val request: SubmitFinalDeclarationRequest = SubmitFinalDeclarationRequest(
       nino,
       taxYear,
-      calculationId
+      CalculationId(calculationId)
```

Comment:
We should update this to follow the same format as nino and taxYear i.e: `val calculationId: CalculationId = CalculationId("a1e8057e-fbbc-47a8-a8b4-78d9f015c253")`

```suggestion
      calculationId
```